### PR TITLE
DGC documentation

### PR DESCRIPTION
**Pull request by AliceBilc**
## Description
DGC documentation.

### CODE REVIEW

File: docs/DISTRIBUTION.md [hunk 1]
```diff
@@ -257,3 +257,23 @@ adding the rolling period of the key to its start interval number).
...
+## Digital Green Certificate
+
+A Digital Green Certificate is a digital proof that a person has either been vaccinated against COVID-19, received a negative test result or recovered from COVID-19.
+
+There are three different value sets to consume:
+ - vaccine-prophylaxis.json - Vaccine or prophylaxis
+ - vaccine-medicinal-product.json - Vaccine medicinal product
+ - vaccine-mah-manf.json - Marketing Authorization Holder
+
+They can be found in the dgc folder. ([here](https://github.com/corona-warn-app/cwa-server/tree/5e47a2e485585043a05ec4173204dd020c757585/services/distribution/src/main/resources/dgc))
```
Comment (roesslerj) [inline docs/DISTRIBUTION.md:270]:
```suggestion
They can be found in the [dgc folder](https://github.com/corona-warn-app/cwa-server/tree/5e47a2e485585043a05ec4173204dd020c757585/services/distribution/src/main/resources/dgc).
```

File: docs/DISTRIBUTION.md [hunk 2]
```diff
@@ -257,3 +257,23 @@ adding the rolling period of the key to its start interval number).
...
+
+The [`DigitalGreenCertificateToProtobufMapping`](/services/distribution/src/main/java/dgc/coronawarn/server/services/distribution/dgc/DigitalGreenCertificateToProtobufMapping.java)
+is responsible for reading the values, using the [`DistributionServiceConfig`](/services/distribution/src/main/java/dgc/coronawarn/server/services/distribution/config/DistributionServiceConfig.java) and transforming the files into protobuffs.
+
+At the end of the process there is created this structure: ehn-dgc/{supportedLanguage}/value-sets.
```
Comment (roesslerj) [inline docs/DISTRIBUTION.md:275]:
```suggestion
At the end of the process these URLs are created to allow retrieving the protobuf files: `ehn-dgc/{supportedLanguage}/value-sets`.
```

File: docs/DISTRIBUTION.md [hunk 3]
```diff
@@ -257,3 +257,23 @@ adding the rolling period of the key to its start interval number).
...
+
+The supported languages are configurable, for now: supportedLanguage: DE, EN, BG, PL, RO, TR. (more details [here](https://github.com/corona-warn-app/cwa-server/blob/5e47a2e485585043a05ec4173204dd020c757585/services/distribution/src/main/resources/application.yaml#L208)).
```
Comment (roesslerj) [inline docs/DISTRIBUTION.md:277]:
```suggestion
The supported languages are [configurable](https://github.com/corona-warn-app/cwa-server/blob/5e47a2e485585043a05ec4173204dd020c757585/services/distribution/src/main/resources/application.yaml#L208), for now they are: DE, EN, BG, PL, RO, TR.
```

**Review by roesslerj — CHANGES_REQUESTED** [review]
All of these are merely suggestions

File: docs/DISTRIBUTION.md [hunk 4]
```diff
@@ -257,3 +257,23 @@ adding the rolling period of the key to its start interval number).
...
 until the threshold minimum is fulfilled.
+
+## Digital Green Certificate
+
+A Digital Green Certificate is a digital proof that a person has either been vaccinated against COVID-19, received a negative test result or recovered from COVID-19.
+
+There are three different value sets to consume:
+ - vaccine-prophylaxis.json - Vaccine or prophylaxis
+ - vaccine-medicinal-product.json - Vaccine medicinal product
+ - vaccine-mah-manf.json - Marketing Authorization Holder
```
Comment (ioangut) [inline docs/DISTRIBUTION.md:268]: Before each of these bullets point you would need to remove the whitespace.:)

**Review by roesslerj — APPROVED** [review]

**Review by ioangut — APPROVED** [review]

**Pull request merged**